README.md

```diff
@@ -53,12 +53,13 @@ diff("a", "a") is None
 
 ---
 
-The diff will treat lists, sets and tuples as lists which means that the result will contain lists instead of sets or tuples.
+The diff will treat lists, sets and tuples as lists, which means that the result will contain lists instead of sets or tuples.
```

Comment:
I would also like to have an example for this, because to me this sentence alone doesn't fully convey what you mean.

Reply:
Opened #5 for this :)